script/Ciel/Day 13/13_01_01_CIEL13_3B.txt

```diff
@@ -1409,12 +1514,13 @@ It was there that the girl learned what she had become."
 [sha:44f90b772409480e64c17318846e0ee38f3761a1]{
 -- Page 227, Offset 33830.
 -- 「……まあ、ようするにその子は矛盾しているんです。
-"...Well, to put it simply, she was a contradiction.
+"Well, to put it simply, she was a contradiction.
 }
 [sha:f39fc74001750f57806e8a05578afa21acce228b]{
 -- Page 227, Offset 33831.
 -- その子はロアとして生まれた人間です。たとえ12歳までの人格が少女のものであっても、魂の名前はロアなんです。
-She was a human who was born as Roa. Even though she had her own personality until she turned twelve, the name of her soul was still that of Roa.
+She was a human who was born as Roa.
+ Even if she had her own personality until she turned twelve, the name of her soul was still that of Roa.
 }
 [sha:0627652426f0ed66e52785b6a22d6abf87eb6c19]{
 -- Page 227, Offset 33832.
```

Comment:
I think this is still part of the explanation they gave to her in the past, so it should be in past tense, I'd figure?

Might be the same for the rest of the upcoming lines? Since she's describing something people told her in the past.